tests/test_grid.py

```diff
@@ -6,103 +6,86 @@
 DESCRIPTION
 This module tests the following functions from grid.py:
 
-- class CoreGridConfig:
-    Configure the `core.grid` module.
-    This data class is used to setup the arrangement of grid cells to be used in
-    running a `virtual_rainforest` simulation.
-
-- CoreGrid._make_square_grid(cell_area: float, cell_nx: int, cell_ny: int) -> dict:
+- grid.make_square_grid
     Create a square grid.
 
-- CoreGrid._make_hex_grid(cell_area: float, cell_nx: int, cell_ny: int) -> dict:
+- grid.make_hex_grid
     Create a hexagonal grid.
 
-- TODO CoreGrid.export geojson
-    Export grid system as geojson file.
-    outfile: A self.cell_dict returned by a make_square_grid or make_hex_grid call.
+- TODO The grid.Grid class.
 
-- TODO Once it is extracted from where it is, _feature.
+- TODO grid.Grid.dump and grid.Grid.dumps for0 geojson
 
 """
 
+import numpy as np
+import numpy.linalg as LA
 import pytest
 from hypothesis import given, settings
 from hypothesis.strategies import integers
+from scipy.spatial.distance import euclidean  # type: ignore
 
-
-def test_CoreGridConfig():
-    """Test if CoreGridConfig().
-
-    Check if function created an object with default values for the following
-    properties: grid_type, cell_contiguity, cell_area, cell_nx, cell_ny.
-    """
-    from virtual_rainforest.core.grid import CoreGridConfig
-
-    obj_1 = CoreGridConfig()
-    assert obj_1.grid_type == "square"
-    assert obj_1.cell_contiguity == "rook"
-    assert obj_1.cell_area == 100
-    assert obj_1.cell_nx == 10
-    assert obj_1.cell_ny == 10
+from virtual_rainforest.core.grid import make_hex_grid, make_square_grid
 
 
 @settings(deadline=None)
 @given(integers(min_value=0, max_value=9), integers(min_value=0, max_value=9))
 def test_make_square_grid(x, y):
     """Test make_square_grid()."""
-    from scipy.spatial.distance import euclidean  # type: ignore
-
-    from virtual_rainforest.core.grid import CoreGrid
 
-    test = CoreGrid._make_square_grid(100, 10, 10)
+    test = make_square_grid(cell_area=100, cell_nx=10, cell_ny=10)
 
-    # check if correct number of points were created
+    # check if correct number of cells were created
     assert len(test.keys()) == 100
 
-    # check distance between corner points is the same
-    distance = euclidean(test[f"{x}-{y}"]["poly"][0], test[f"{x}-{y}"]["poly"][2])
-    distance2 = euclidean(test[f"{x}-{y}"]["poly"][1], test[f"{x}-{y}"]["poly"][3])
-    assert distance == pytest.approx(distance2)
+    # check number of vertices in outer ring is correct
+    xy = np.array(test[f"{x}-{y}"].exterior.coords)
+    assert xy.shape == (5, 2)
+
+    # check that distance between corner points is the same
+    # fmt: off
+    distance = euclidean(xy[0, ], xy[2, ])
+    distance2 = euclidean(xy[1, ], xy[3, ])
```

Comment:
This looks odd. Why not just do the following and remove the skip formatting tags? The result is the same, and seems a lot cleaner. 
```suggestion
    distance = euclidean(xy[0], xy[2])
    distance2 = euclidean(xy[1], xy[3])
```
And the same below.

Reply:
So it is! That's an R/Python thing. If you give an R matrix a single index, then it is an index to a single element, iterating over the axes. I'd forgotten that it could return a slice in Python.